ChustaSoft.Tools.Authorization.TestBasic.WebAPI/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "AuthorizationConnection": ""
+    "AuthorizationConnection": "Server=myServerName\nyInstanceName;Database=nyDataBase;User Id=myUsername;Password=myPassword;"
```

Comment:
Remove it before PR integration, better leave the user to directly paste everything instead of override settings one by one